app/services/testdata/GeneratorConfig.scala

```diff
@@ -20,20 +20,22 @@ import model.EvaluationResults.Result
 import org.joda.time.{ DateTime, LocalDate }
 
 case class GeneratorConfig(emailPrefix: String,
-                           setGis: Boolean = false,
-                           cubiksUrl: String,
-                           firstName: Option[String],
-                           lastName: Option[String],
-                           preferredName: Option[String],
-                           isCivilServant: Option[Boolean],
-                           hasDegree: Option[Boolean],
-                           region: Option[String],
-                           loc1scheme1Passmark: Option[Result],
-                           loc1scheme2Passmark: Option[Result],
-                           previousStatus: Option[String],
-                           confirmedAllocation: Boolean = true,
-                           dob: Option[LocalDate],
-                           postCode: Option[String],
-                           country: Option[String],
-                           phase1StartTime: Option[DateTime],
-                           phase1ExpiryTime: Option[DateTime])
+  setGis: Boolean = false,
+  cubiksUrl: String,
+  firstName: Option[String],
+  lastName: Option[String],
+  preferredName: Option[String],
+  isCivilServant: Option[Boolean],
+  hasDegree: Option[Boolean],
+  region: Option[String],
+  loc1scheme1Passmark: Option[Result],
+  loc1scheme2Passmark: Option[Result],
+  previousStatus: Option[String],
+  confirmedAllocation: Boolean = true,
+  dob: Option[LocalDate],
+  postCode: Option[String],
+  country: Option[String],
+  phase1StartTime: Option[DateTime],
+  phase1ExpiryTime: Option[DateTime],
+  tscore: Option[Double]
```

Comment:
Talked about this, could potentially be used for phase1 and phase2 (if generating a phase2 test phase1 is implied as passed etc)